services/server/config/config.go

```diff
@@ -255,25 +255,30 @@ func loadConfigFile(path string) (*Config, error) {
 // - Non abs path is relative to parent config file directory
 // - Abs paths returned as is
 func resolveImports(parent string, imports []string) ([]string, error) {
-	var out []string
+	paths := make(map[string]struct{})
 
 	for _, path := range imports {
+		path = filepath.Clean(path)
+		if !filepath.IsAbs(path) {
+			path = filepath.Join(filepath.Dir(parent), path)
+		}
 		if strings.Contains(path, "*") {
 			matches, err := filepath.Glob(path)
 			if err != nil {
 				return nil, err
 			}
 
-			out = append(out, matches...)
-		} else {
-			path = filepath.Clean(path)
-			if !filepath.IsAbs(path) {
-				path = filepath.Join(filepath.Dir(parent), path)
+			for _, match := range matches {
+				paths[filepath.Clean(match)] = struct{}{}
```

Comment:
The path operations are handled with the filepath function, so it looks like there's no need to use `filepath.Clean`